scenarios/mqtt-pravega-bridge/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.4'
+}
+
+group 'com.dell.mqtt'
```

Comment:
all of the samples are using `io.pravega.example.` package structure. Could you please do the same for this sample app too?

Reply:
Fixed